#474: Converted test api test environment docker runtime to pytest (#481)
exasol/integration-test-docker-environment: doc/user_guide/user_guide.rst

select a random free port.

Docker Runtimes
~~~~~~~~~~~~~~~~~~~~~~

> ITDE supports the launch of the test environment with a specified
> docker runtime, via option ``--docker-runtime``.

Reviewer: ```suggestion
ITDE supports launching of the test environment with an alternate
docker runtime, via option ``--docker-runtime``.
```

> A container runtime is essential software
> enabling containers to function within a host environment.
> It handles tasks ranging from retrieving container images
> from a registry and managing their lifecycle
> to executing the containers on your system.
> See https://docs.docker.com/engine/daemon/alternative-runtimes/ for details.

Reviewer: ```suggestion
The docker runtime is the software enabling containers to function within a
host environment.  It handles tasks ranging from retrieving container images
from a registry and managing their lifecycle to executing the containers on
your system.  See https://docs.docker.com/engine/daemon/alternative-runtimes/
for details.
```
